internal/etcd/etcd.go

```diff
@@ -0,0 +1,263 @@
+package etcd
+
+import (
+	"bytes"
+	"encoding/json"
+	"golang.org/x/net/context"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/BurntSushi/toml"
```

Comment:
do we need to use BurntSushi/toml? Can you find a way to use influxdata/config or naoina/toml instead?